gui/src/app/init/topology/relation.js

```diff
@@ -31,9 +31,10 @@ const RelationMenu = require('../../components/relation-menu/relation-menu');
  * @class RelationModule
  */
 class RelationModule {
-  constructor() {
+  constructor(options={}) {
```

Comment:
driveby fix?

Reply:
This is require to pass in the analytics instance below (`this.analytics = options.analytics.addCategory('Relation');`).